### ISSUE TITLE
Unable to locate libpython3.so message

### ISSUE THREAD

**Issue by zzztimbo** [polynote/polynote#522]
I'm using `pyenv`. I can only get Python working if I use the `system` version in `pyenv`.

**Comment by jeremyrsmith** [comment]
Welcome, thanks for the error report!

I assume you're getting this message from the run script? It doesn't do a very good job of finding the python library; we'll make this configurable with an environment variable (maybe `pyenv` sets one that we can look at?)

Currently it just looks for the python library that sits a couple directories up from where `jep` is installed. I think it probably has to use the same python that jep was built for in any case.

Does `pip3 show jep` point to the right python installation?

**Comment by zzztimbo** [comment]
hey @jeremyrsmith the `pip3 show jep` reveals location: `/Users/timchan/Downloads/polynote/.venv/lib/python3.7/site-packages`. 

I read issue https://github.com/polynote/polynote/issues/521 and was able to get Python working with the suggestion you posted in that issue. 

This worked for me: 
```
export POLYNOTE_DEPS=($(ls deps/*.jar))
java -cp polynote.jar:$(IFS=: ; echo "${POLYNOTE_DEPS[*]}") -Djava.library.path=/Users/timchan/Downloads/polynote/.venv/lib/python3.7/site-packages/jep polynote.Main
```

I was able to execute a Python cell, but now I have issues with dependencies. I've added `pandas` as a dependency, but an `import pandas as pd` gives me `Uncaught exception: ModuleNotFoundError: No module named 'pandas' (java.lang.RuntimeException)`
